src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java

```diff
@@ -58,19 +58,36 @@ public boolean hasResponseFullAdmit() {
 
     @Override
     public boolean defendantRespondedWithFullAdmitAndPayImmediately() {
+        if (isClaimProceedInCaseMan()) {
+            return false;
+        }
         return hasResponseFullAdmit()
-            && isPayImmediately();
+                && isPayImmediately();
+    }
+
+    private boolean isClaimProceedInCaseMan() {
```

Comment:
Move to parent class to avoid duplicating.

Reply:
updated